homeassistant/components/cookidoo/strings.json

```
@@ -9,8 +9,8 @@
           "country": "Country"
         },
         "data_description": {
-          "email": "Email used access your Cookidoo account.",
-          "password": "Password used access your Cookidoo account.",
+          "email": "Email used to access your Cookidoo account.",
```

Could it make sense to use placeholders?
```suggestion
          "email": "Email used to access your {cookidoo} account.",
```

---

@epenet What is a placeholder used for?

---

It is used to pass in dynamic non-translatable content, so that translators can ignore it.
It is passed in `async_show_form` like this:
```python
    return self.async_show_form(
            step_id="user",
            data_schema=schema,
            description_placeholders={"cookidoo": "Cookidoo"},
        )
```

---

Ahh, sorry @epenet I was unsure if I understood correctly, because I already used placeholder but not to "avoid" translations, more for dynamic values. But this looks elegant!

---

@epenet I updated it everywhere in the config_flow (data, data_description, title).